src/main/scala/is/hail/HailContext.scala

```diff
@@ -194,14 +194,16 @@ class HailContext private(val sc: SparkContext,
   def importBgen(file: String,
     sampleFile: Option[String] = None,
     tolerance: Double = 0.2,
-    nPartitions: Option[Int] = None): GenericDataset = {
-    importBgens(List(file), sampleFile, tolerance, nPartitions)
+    nPartitions: Option[Int] = None,
+    gr: GenomeReference = GenomeReference.defaultReference): GenericDataset = {
```

Comment:
I think most of these methods will also need to be updated in the Python calls since the method signatures have changed, right?

Reply:
I had originally split the scala and python changes into 2 PRs. Now they're all in this PR. Scala is 1st commit and Python is 2nd commit. 

@cseed can you please look over the default reference structure to make sure I addressed your comment from #2096.

Reply:
Few small comments but this is almost exactly what I was envisioning.  Nice work.